oxigen_stock_barcodes_gs1/wizard/stock_barcodes_read.py

```diff
@@ -14,13 +14,31 @@ class WizStockBarcodesRead(models.AbstractModel):
 
     def process_lot(self, barcode_decoded):
         res = super().process_lot(barcode_decoded)
+        if self.product_id.tracking == "serial":
+            lot_barcode = barcode_decoded.get("21", False)
+            lot = self.env["stock.production.lot"].search(
+                [("name", "=", lot_barcode), ("product_id", "=", self.product_id.id)]
```

Comment:
add "company_id" but not using self.env.company, using the company of the current document, picking, inventory, etc..